Translator Release Process Details

```diff
@@ -0,0 +1,75 @@
+TACT decides on a case-by-case basis if production deployment is go/no-go for new component(s)
+TACT votes via procedures outlined in the current Consortium Guidelines - see Change Management section here
+Subsequent Release Planning
+Team/Working Group/Committee/External Group bring proposal(s) for new component(s) or extensive updates to existing component(s) to TACT for review
+Teams to include the following details: 
+Theme (required) 
+e.g. what working group, committee, architecture component this task belongs to.
+Milestone: a short, executive summary of the proposed work. (required)
+e.g. - Fix Biolink Documentation: Deliver a refactored repository following LinkML best practices with documentation that is searchable and navigable as it previously was.
+e.g. - Host a series of discussions around modeling “variants” for use in Translator: At the end of the cycle, propose options to the Data Modeling team for confirmation. 
+e.g. - Revisit SRI confidence score [g() function]: deploy a revised scoring algorithm that takes SRI confidence score into account.
+Specific steps to accomplish the milestone (or a link to a GitHub project tracker) (required)
+Timeline for completing the proposed work for completing the proposed work
+Responsible teams (or individuals) (required)
+Status (optional)
+Any additional notes that help clarify the tasks represented by the deliverable (optional)
+e.g. - will this involved release coordination with any other teams? 
+e.g. - will the release of this component mean work for another team next release? 
+Example: winter 2023/4 release milestones reviewed by TACT
+TACT defines the work units required for production deployment of the proposed component(s) as part of the next release.
+
+Review process, revise if necessary, and repeat as part of next release
+GOAL: formalize the need to not institutionalize processes that make production of Translator components unnecessarily difficult, slow, confusing, or monotonous.
+Retrospective participants should feel comfortable expressing their successes and challenges in the context of the release, bringing with them specific examples (when possible) of things that were good or bad, preferably with suggestions on how to make the process better.  
+Hotfixes
+GOAL: Allow changes to go into PROD during the coding window when they address critical errors in the deployed feature.
+Definition: a hotfix is a code change to address a bug that is deployed between official release cycles. Hotfixes do not address non-critical issues, new features, or incremental improvements and optimizations.
+Policy: Teams can propose hotfixes to be reviewed and approved by TACT
+TACT members making hotfix decisions, will greatly benefit from hotfix descriptions following a basic template:
+Sample Changelog for Code Window deployments to PROD
+sample changelog for PROD deployment request - ARAX
+
+Translator Release Schedule Timeline
+Translator Release Process Guidelines
+
+
+Nomenclature:
```

Comment:
The heading "Nomenclature" makes me think that what follows will be a glossary, but instead it seems like some residual internal notes